style2.css

```diff
@@ -0,0 +1,128 @@
+*{
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+.title {
+  padding: 2%;
+  text-align: center;
+}
+
+#check-menu {
+  display: none;
+}
+
+.menu {
+  background: #b39ddb;
+  display: block;
+  float: right;
+  height: 100vh;
+  width: 30%;
+}
+
+.menu__link {
+  display: block;
+  width: 100%;
+  padding: 5%;
+  background-color: #b39ddb;
+  color: white;
+  text-decoration: none;
+  border-bottom: .2px solid #e0e0e0;
+  text-align: center;
+  font-size: 1.2em;
+}
+
+.menu__link:hover {
+  background: grey;
+}
+
+        /* media */
+
+@media (max-width:425px) {
+  .wrapper {
+    background-color: #ede7f6;
+    position: relative;
+    height: 40px;
+  }
+  
+  #check-menu {
+    display: none;
+  }
+  
+  label {
+    display: block;
+    position: absolute;
+    width: 100%;
+    height: 100%;
+    cursor: pointer;
+    z-index: 2;
+    line-height: 40px;
+    margin-right: 0;
+  }
+
+  .burger-line {
+    position: absolute;
+    top: 8px;
+    right: 10px;
+    width: 30px;
+    height: 2px;
+    background-color: white;
+    transition: .3s all;
+  }
+
+  .second {
+    top: 18px;
+  }
+
+  .third {
+    top: 28px;
+  }
+
+  .menu {
+    display: block;
+    position: absolute;
+    width: 100%;
+    float: right;
+    top: 40px;
+    right: 0;
+    transform: translateX(50%);
+    max-width: 0;
+    transition: .5s all;
+  }
+
+  .menu__link {
+    display: block;
+    width: 50%;
+    padding: 2%;
+    background-color: #b39ddb;
+    color: white;
+    text-decoration: none;
+    border-bottom: .2px solid #e0e0e0;
+    text-align: center;
+    padding-left: 5%;
+  }
+
+  .menu__link:hover {
+    background: grey;
+  }
+
+  #check-menu:checked ~ .menu {
+    max-width: 70%;
+   
+  }
+
+  #check-menu:checked ~ .first {
+    transform: rotate(45deg);
+    transform-origin: left top;
+  }
+
+  #check-menu:checked ~ .second {
+    height: 0;
+  }
+
+  #check-menu:checked ~ .third {
+    transform: rotate(-45deg);
+    transform-origin: left bottom;
+  }
+}
```

Comment:
Codacy found an issue: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/WebHeroSchool/css17__/pullRequest?prid=5991486)